### PR TITLE
contrib/qemu: enable liburine, libssh, capstone

### PR DESCRIPTION
**Pull request by nekopsykose**

### CODE REVIEW

**Comment by wezm** [comment]
Was really wondering what on earth liburine was but then saw it was a liburing typo :sweat_smile:

**Comment by q66** [comment]
it's a more appropriate name i feel

File: contrib/libssh/template.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "pkgconf",
+]
+makedepends = [
+    "cmocka-devel",
+    "openssl-devel",
+    "zlib-devel",
+]
+pkgdesc = "Library for accessing ssh client services through C libraries"
+maintainer = "psykose <alice@ayaya.dev>"
+license = "( LGPL-2.1-or-later WITH custom:openssl-exception ) AND BSD-2-Clause"
```
Comment (q66) [inline contrib/libssh/template.py:21]: drop the parens

File: contrib/libssh/template.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+url = "https://www.libssh.org"
+source = f"https://www.libssh.org/files/{pkgver[:pkgver.rfind('.')]}/libssh-{pkgver}.tar.xz"
+sha256 = "b60e2ff7f367b9eee2b5634d3a63303ddfede0e6a18dfca88c44a8770e7e4234"
+hardening = ["vis", "cfi"]
+
+
+def post_install(self):
+    self.install_license("COPYING")
```
Comment (q66) [inline contrib/libssh/template.py:29]: BSD should also be installed

**Pull request merged**